teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+
+# class Teacher
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] The `Teacher` class inherits from the `Person` class which means that `parent_permission` should be included in the constructor as well, with a default value.

Reply:
There why I left the parent_permission from getting inherited by the Teacher class from the Person class is because the default value is true (in Person class) and I don't think parent_permission is necessary for teachers for they are adults themselves.